README.rst

```diff
@@ -116,6 +119,52 @@ Usage
       latest     Get the latest tag in a changelog.
       tag        Create an annotated git tag based on a changelog entry.
       update     Updates the version and date of the most recent changelog entry.
+      create     Creates a new CHANGELOG.md file with a standard header.
+      entry      Add a quick entry to the current version.
+      next       Create a new unreleased version in changelog file.
+
+next
+----
+
+Create a new unreleased version in changelog file.
+
+::
+
+   Usage: chag next
+
+   Creates a new unreleased version inside of the changelog file.
+
+   Options:
+     --help     Displays this message.
+
+entry
```

Comment:
I thought your changes are worth to be pulled into my fork since I use them. I did not know you would not like that to happen and apologize for doing so. Rebasing and rewriting history regularly is not a trivial task and I like to save time where ever I can for stuff that has greater importance to me. So there's only one option that I can do at this point. Would you like me to back out all of your new commands from my fork?